askem.dbml

```diff
@@ -15,11 +15,29 @@ Enum public.resource_type {
 Enum public.provenance_type {
    Dataset
    Model
+   ModelParameter
    Plan
+   PlanParameter
    ModelRevision
    Intermediate
    Publication
    SimulationRun
+   Project
+   Concept
+}
+
+Enum provenance_abbreviations{
+  Ds 
+  Md
+  Mp
+  Sp 
+  Pp
+  Mr
+  In
+  Pu
+  Sr
+  Pr
+  Cn
 }
 
 Enum provenance_search_types{
```

Comment:
Do we want this in the dbml?

Reply:
I moved it out. Doesn't need to be there